### ISSUE TITLE
Feature request: OsmAnd offline maps (.obf) as map background

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MarcusWolschon/osmeditor4android#272]
```
I collect much mapping information in areas that don't offer internet for long 
periods.

I would like to use offline maps to be able to prepare an edit while offline. 
Preferably .obf format as I use OsmAnd for navigation and want to avoid 
downloading the same map information twice in a different format.
```

Original issue reported on code.google.com by `jan.vanb...@gmail.com` on 16 Feb 2015 at 4:24
- Blocked on: #118
- Merged into: #118
